src/components/NotesPage/NotesPageActions.js

```diff
@@ -0,0 +1,5 @@
+export const CHANGE_HELP_VIEW_ACTION  = 'CHANGE_HELP_VIEW_ACTION';
```

Comment:
"Change help view" sounds ambiguous.

We show help to the user.
We store "user has seen help".
We don't how help again.

So the action is "user saw help" or "user viewed help".